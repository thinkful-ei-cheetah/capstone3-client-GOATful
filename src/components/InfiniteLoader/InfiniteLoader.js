import React from 'react'
import './InfiniteLoader.css'

export default function SmallLoadingAnimation() {
  return (
    <div class="loader-ellips">
  <span class="loader-ellips__dot"></span>
  <span class="loader-ellips__dot"></span>
  <span class="loader-ellips__dot"></span>
  <span class="loader-ellips__dot"></span>
</div>
  )
}
