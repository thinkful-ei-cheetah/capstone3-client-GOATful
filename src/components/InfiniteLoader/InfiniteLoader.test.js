import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import InfiniteLoader from './InfiniteLoader';

describe('InfiniteLoader Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(< InfiniteLoader />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<InfiniteLoader/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
})