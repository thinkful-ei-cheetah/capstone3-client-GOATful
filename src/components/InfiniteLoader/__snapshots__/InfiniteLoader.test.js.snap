// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfiniteLoader Component renders the UI as expected 1`] = `
<div
  className="loader-ellips"
>
  <span
    className="loader-ellips__dot"
  />
  <span
    className="loader-ellips__dot"
  />
  <span
    className="loader-ellips__dot"
  />
  <span
    className="loader-ellips__dot"
  />
</div>
`;
